src/pin-field/PinField.ts

```diff
@@ -66,6 +66,12 @@ export class PinField extends OmniFormElement {
      */
     @property({ type: Boolean, reflect: true, attribute: 'no-native-keyboard' }) noNativeKeyboard?: boolean;
 
+    /**
+     * Adds character limit for the pin input field.
+     * @attr [max-length]
+     */
+    @property({ reflect: true, attribute: 'max-length' }) maxLength?: number;
```

Comment:
Advise that you add the type as per suggestion below.

```suggestion
    @property({ type: Number,  reflect: true, attribute: 'max-length' }) maxLength?: number;
```

Reply:
The Property Type has been added.